recipes/setup_server.rb

```diff
@@ -28,4 +28,17 @@
     notifies :restart, "service[apache2]"
 end
 
+template "/etc/init.d/websockify" do
+    source "websockify.erb"
+    owner "root"
+    group "root"
+    variables({
+        :websockify_port => node['abiquo']['websockify']['port'],
+        :websockify_key => node['abiquo']['websockify']['key'],
+        :websockify_crt => node['abiquo']['websockify']['crt']
+    })
+    action :create
+    notifies :restart, "service[websockify]"
```

Comment:
Tests fail now because the "websockify service" has not been declared and Chef does not know what it is. You need to create a "service 'websockify'" to let Chef know of its existance.